SU2_CFD/src/numerics_direct_mean.cpp

```diff
@@ -2964,6 +2964,13 @@ void CUpwRoe_Flow::ComputeResidual(su2double *val_residual, su2double **val_Jaco
   if (roe_low_dissipation)
     SetRoe_Dissipation(Dissipation_i, Dissipation_j, Sensor_i, Sensor_j, Dissipation_ij, config);
   
+  su2double dissipationVec[5];
+  dissipationVec[0] = Dissipation_ij;
+  dissipationVec[1] = Dissipation_ij;
+  dissipationVec[2] = Dissipation_ij;
+  dissipationVec[3] = Dissipation_ij; //max(Dissipation_ij, 0.1);
+  dissipationVec[4] = Dissipation_ij; //max(Dissipation_ij, 0.1);
+
```

Comment:
What's going on here? I'm guessing you were trying different central/upwind weightings for different flow equations?

Reply:
Yes, you're right, but it did not solve the problem I was trying to get at, so we can take it out.  At the moment it is just clutter.